Can't build with mingw-w64 gcc 10

```
creating Makefile

current directory: C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6/ext/certstore
make "DESTDIR=" clean

current directory: C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6/ext/certstore
make "DESTDIR="
generating certstore-x64-mingw32.def
compiling certstore.c
compiling certstore_util.c
compiling loader.c
linking shared-object certstore/certstore.so
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x0): multiple definition of `rb_eCertLoaderError'; certstore.o:certstore.c:(.bss+0x0): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x8): multiple definition of `rb_cCertLoader'; certstore.o:certstore.c:(.bss+0x8): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: certstore_util.o:certstore_util.c:(.bss+0x10): multiple definition of `rb_mCertstore'; certstore.o:certstore.c:(.bss+0x10): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x0): multiple definition of `rb_eCertLoaderError'; certstore.o:certstore.c:(.bss+0x0): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x8): multiple definition of `rb_cCertLoader'; certstore.o:certstore.c:(.bss+0x8): first defined here
C:/ruby26/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/10.1.0/../../../../x86_64-w64-mingw32/bin/ld.exe: loader.o:loader.c:(.bss+0x10): multiple definition of `rb_mCertstore'; certstore.o:certstore.c:(.bss+0x10): first defined here
collect2.exe: error: ld returned 1 exit status
make: *** [Makefile:261: certstore.so] Error 1

make failed, exit code 2
Gem files will remain installed in C:/ruby26/lib/ruby/gems/2.6.0/gems/certstore_c-0.1.6 for inspection.
Results logged to C:/ruby26/lib/ruby/gems/2.6.0/extensions/x64-mingw32/2.6.0/certstore_c-0.1.6/gem_make.out
```

--- a/ext/certstore/certstore.c
+++ b/ext/certstore/certstore.c
@@ -11,6 +11,8 @@
 
 #include <certstore.h>
 
+VALUE rb_mCertstore;
+
 void
 Init_certstore(void)
 {

--- a/ext/certstore/certstore.h
+++ b/ext/certstore/certstore.h
@@ -31,11 +31,11 @@
 
 #include <wincrypt.h>
 
-VALUE rb_mCertstore;
-VALUE rb_cCertLoader;
-VALUE rb_eCertLoaderError;
+extern VALUE rb_mCertstore;
+extern VALUE rb_cCertLoader;
+extern VALUE rb_eCertLoaderError;
 
 void handle_error_code(VALUE self, DWORD errCode);
-void Init_certstore_loader(VALUE rb_mCertstore);
+void Init_certstore_loader(VALUE certstore);
 
 #endif // _WINEVT_H

--- a/ext/certstore/loader.c
+++ b/ext/certstore/loader.c
@@ -11,6 +11,9 @@
 
 #include <certstore.h>
 
+VALUE rb_cCertLoader;
+VALUE rb_eCertLoaderError;
+
 struct CertstoreLoader
 {
   HCERTSTORE hStore;
@@ -502,9 +505,9 @@ error:
 }
 
 void
-Init_certstore_loader(VALUE rb_mCertstore)
+Init_certstore_loader(VALUE certstore)
 {
-  rb_cCertLoader = rb_define_class_under(rb_mCertstore, "Loader", rb_cObject);
+  rb_cCertLoader = rb_define_class_under(certstore, "Loader", rb_cObject);
   rb_eCertLoaderError =
     rb_define_class_under(rb_cCertLoader, "LoaderError", rb_eStandardError);
 

